packages/passport/sdk/src/guardian/guardian.ts

```diff
@@ -155,8 +160,8 @@ export default class GuardianClient {
   private async evaluateEVMTransaction({
     chainId,
     nonce,
-    user,
     metaTransactions,
+    user,
```

Comment:
We can, the reason I passed through the user is because the only call to evaluateEvmTransaction is on the very next line after we retrieve the user in validateEvmTransaction, which is something we need to do in order to provide the address for confirmationScreen.requestConfirmation, so it seemed redundant. Happy to change it for consistency if you'd prefer though.